pyscriptjs/src/pyscript.py

```diff
@@ -68,7 +72,7 @@ def format_mime(obj):
         format_dict, md_dict = mimebundle
     else:
         format_dict = mimebundle
-        md_dict = {}
+        md_dict = {}  # noqa: F841
```

Comment:
`md_dict` can be removed if it's not used. We shouldn't add the `noqa` here.

Reply:
Removed.